ansible/files/awslogs.conf

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
It's not a good practice to copy full config file

Reply:
Made changes in https://github.com/dtrendiuk/learning/pull/4/commits/45d1fd12a5e01fecd2166d99a9c569450bb48f30